packages/design-system/src/styles/components/_Alert.scss

```diff
@@ -51,6 +51,29 @@ $alert-link-color-focus: $color-primary-darkest;
   }
 }
 
+.ds-c-alert__a11y-label {
+  margin-right: 0.5rem;
+
+  /* stylelint-disable */
+  // !important needed to overwrite screen reader utility styles
```

Comment:
It hurt adding these `!important` flags, but because we're using the SR utility class to visually hide the label, there was a specificity issue when targeting this class. And I couldn't target a `<span>` like I did with Badge since theoretically users could insert their own `<span>` as a child - and I don't want to break things for them.

Reply:
This is kind of a tangent, but it's why I've personally moved away from declaring `!important` on utility classes. I like the idea of `!important` forcing immutability, which is why I like keeping it on a per-declaration basis.